### DISCUSSION TITLE
Some questions about PKGBUILD

### DISCUSSION THREAD

**Discussion by ghost** [msys2/MINGW-packages#21130] [Q&A] [answered]
I was attempted create PKGBUILD files and have something trouble about it.
1. Why not apply `-j${nproc}` to `make` command?
2. Why pkgbase uses `mingw-w64-` as prefix?
3. I attempted create a PKGBUILD for a cmake project that no longer maintained, but makepkg complained "==> WARNING: Package contains reference to $(cygpath -m /)". What should I do?

**Comment by Biswa96** [comment, marked as answer]
> 1. Why not apply `-j${nproc}` to `make` command?

Add that flag in `MAKEFLAGS` in `/etc/makepkg.conf` file.

> 2. Why pkgbase uses `mingw-w64-` as prefix?

To distinguish it from msys2 packages.

> 3. makepkg complained "==> WARNING: Package contains reference to $(cygpath -m /)". What should I do?

Use `sed` to replace that full path reference with appropriate keyword or root path - it depends. See other PKGBUILD files to get some idea.
